pilosa.go

```diff
@@ -56,6 +57,14 @@ var (
 	ErrFragmentNotFound = errors.New("fragment not found")
 	ErrQueryRequired    = errors.New("query required")
 	ErrTooManyWrites    = errors.New("too many write commands")
+
+	ErrConfigClusterTypeInvalid = errors.New("invalid cluster type")
+	ErrConfigClusterTypeMissing = errors.New("missing cluster type")
+	ErrConfigHostsMissing       = errors.New("missing bind address in cluster hosts")
+	ErrConfigBroadcastPort      = errors.New("missing broadcast port in internal-hosts")
```

Comment:
I would change this message to something like "internal-port not found in internal-hosts" since there is nothing about a "broadcast port" in the config file.